src/commands/create-avd.yml

```diff
@@ -14,23 +14,23 @@ parameters:
     type: boolean
     description: |
       Whether to first install the system image via sdkmanager
+  background:
+    type: boolean
+    description: |
+      Where to run the creation command in background
```

Comment:
i think it might be confusing if users are wondering "where" rather than "if" the creation command should be run in the background

Reply:
I agree that was some oversight by me, I just fixed it and made a new commit